src/otx/core/model/classification.py

```diff
@@ -711,6 +730,13 @@ def _convert_pred_entity_to_compute_metric(
             "target": target,
         }
 
+    def get_dummy_input(self, batch_size: int = 1) -> MulticlassClsBatchDataEntity:
+        """Returns a dummy input for classification OV model"""
+        # Resize is embedded to the OV model, which means we don't need to know the actual size
+        images = [torch.rand(3, 224, 224) for _ in range(batch_size)]
```

Comment:
Don't we need to consider a dataset having multiple input size?

Reply:
I wouldn't say this is true in production, we're a little bit biased by web-scraped data. Usually, the image source is a camera with a stable resolution. 
Changing the size from the default one makes sense: then resize would be really executed. The resize complexity is proportional to output HW, therefore the input image shape is not that important (it shouldn't be too small though to prevent extensive cache usage). I'll change the size from default to let resize be executed